src/LtiAdvantage.IdentityModel/LtiAdvantage.IdentityModel.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I switched this project to netstandard2.0 because IdentityModel targets netstandard2.0 (https://github.com/IdentityModel/IdentityModel2/blob/master/src/IdentityModel.csproj#L3). I did not target net452 or netstandard1.4 because I'm lazy :) 

Did netstandard2.0 cause a problem for you?

Reply:
That's strange. I'm not sure how I sent a pull request for that patch to you. I only meant to send the path fix. Not the change to the TargetFramework.

For some reason netstandard2.0 wasn't work correctly in my project. I'm assume that's a problem on my end.